prep-javascript-objects/prep-javascript-objects-quiz-notes.md

```diff
@@ -0,0 +1,37 @@
+# prep-javascript-objects-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What data types can object properties hold?
+
+- Describe the syntax (structure) of object-literals in JavaScript.
+
```

Comment:
Don't forget to answer the quiz questions!

Reply:
I just re committed with the quiz answered